spec/features/admin_user/admin_user_home_page_spec.rb

```diff
@@ -1,43 +1,41 @@
 require 'rails_helper'
 
-RSpec.feature "the admin user" do
-  let(:valid_attributes) {
-     {full_name: "Admin Annie",
-      user_name: "admin",
-      password: "password",
-      role: 1,
-      email: "admin@admin.com"}
+RSpec.feature "the admin user dashboard" do
+  let(:admin) { User.create(full_name: "Admin Annie",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.